CHANGELOG.rst

```diff
@@ -14,6 +14,8 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
+* Upgraded many dependencies.
```

Comment:
Should this be preceded with a 1.4.0 section header?

Reply:
More changes are coming for 1.4.0.